spec/Ranges.tex

```diff
@@ -932,7 +932,7 @@ \subsection{Range Slicing}
 \end{rationale}
 
 If the resulting represented sequence cannot be expressed as a range of the
-expected type, an error is generated. This can happen, for example, when
+expected type, the empty range \chpl{1..0} is computed as the result. This can happen, for example, when
```

Comment:
You know that LaTeX ignores whitespace, right?  :)  I was just trying to minimize the diff.